deployments/terraform/modules/sqs/queue/main.tf

```diff
@@ -29,7 +29,30 @@ resource "aws_cloudwatch_metric_alarm" "number-of-visible-messages" {
   evaluation_periods  = var.alarm_evaluation_periods
   period              = var.alarm_period
   statistic           = "Maximum"
-  threshold           = var.alarm_threshold
+  threshold           = var.alarm_visible_messages_threshold
+  treat_missing_data  = "notBreaching"
+  datapoints_to_alarm = var.alarm_datapoints_to_alarm
+
+  dimensions = {
+    QueueName = aws_sqs_queue.main.name
+  }
+
+  alarm_actions = ["arn:aws:sns:eu-central-1:164105964448:${var.project}-${var.environment}-${var.family}-alarm"]
+  ok_actions    = ["arn:aws:sns:eu-central-1:164105964448:${var.project}-${var.environment}-${var.family}-alarm"]
+}
+
+resource "aws_cloudwatch_metric_alarm" "oldest_message_age" {
```

Comment:
i would go without the alarm actions for the new one to run a test first..
we don't want to trigger alarms here to annoy everyone.. :D